app/services/hearing_events_recorder.rb

```diff
@@ -7,11 +7,11 @@ def initialize(hearing_id:, hearing_date:, body:)
   end
 
   def call
-    hearing_event_recording.update(body: body)
+    hearing_event_recording.update!(body: body)
```

Comment:
This will raise an exception. Need to confirm with the standard if that is expected.